README.md

```diff
@@ -1,2 +1,97 @@
-# Hello---micro
-First project with microverse
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
```

Comment:
- [ ] The template for the readme file to be used is as follows, [follow this link](https://github.com/microverseinc/readme-template/blob/master/README.md).
- [ ] Copy the template by clicking the two boxes in the image below once you go to the link above.
![image](https://user-images.githubusercontent.com/81932478/170239755-879ed462-18d8-4992-93d3-cdc907991a01.png)

- [ ] Then clear your readme file and paste the template in your readme file on your macjine.
- [ ] Lastly edit the readme file by adding project name, git hub link and linkedin link, as well as the issues link for your current project.